lang/cs/Org.Apache.REEF.Network/StreamingCodec/IDataCodecObj.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Org.Apache.REEF.Wake.Remote;
+
+namespace Org.Apache.REEF.Network.StreamingCodec
+{
+    /// <summary>This is the Codec class that encoded and decodes to the type object 
+    /// rather than some generic type T. This class is mainly built for Group Communication 
+    /// GroupCommunicationMessage cannot have Generic type and hence codec of this type
+    /// needs to be passed. The codecs there will implement this interface</summary>
+    public interface IDataCodecObj
```

Comment:
Should it be called IObjCodec? Sice it is a Codec, usually its name ends with Codec.